### PR TITLE
Added Apache TomEE artifacts

### PR DESCRIPTION
**Pull request by AndyGee**
Also added cross platform attributes

### CODE REVIEW

File: microprofile-sample-swagger/src/main/java/io/microprofile/sample/swagger/utils/QLogger.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * Tomitribe Confidential
+ * <p/>
+ * Copyright(c) Tomitribe Corporation. 2014
+ * <p/>
+ * The source code for this program is not published or otherwise divested
+ * of its trade secrets, irrespective of what has been deposited with the
+ * U.S. Copyright Office.
+ * <p/>
+ */
```
Comment (karianna) [inline microprofile-sample-swagger/src/main/java/io/microprofile/sample/swagger/utils/QLogger.java]: I'm concerned by this license header.  I think depending on Antonio's decision (not sure why he's not a member on this project) we are aiming to Apache License v2 everything.

Comment (AndyGee) [inline microprofile-sample-swagger/src/main/java/io/microprofile/sample/swagger/utils/QLogger.java]: Oops, that's my IDEA default, I'll update to Apache ;-)

**Comment by karianna** [comment]
Looks fine to me but I don't have merge rights.

**Comment by dblevins** [comment]
Added you to the “Owners” section of the MicroProfile org in Github.  You should be able to do anything now :)

**Pull request merged**